jwst/flatfield/tests/test_flatfield.py

```diff
@@ -252,9 +251,9 @@ def test_nirspec_fs_flat(srctype):
     data.slits.append(datamodels.SlitModel(shape))
     data.slits[0].data = np.full(shape, 1.0)
     data.slits[0].dq = np.full(shape, 0)
-    data.slits[0].err = np.full(shape, 0.0)
-    data.slits[0].var_poisson = np.full(shape, 0.0)
-    data.slits[0].var_rnoise = np.full(shape, 0.0)
+    data.slits[0].err = np.full(shape, 0.1)
+    data.slits[0].var_poisson = np.full(shape, 0.01)
+    data.slits[0].var_rnoise = np.full(shape, 0.01)
```

Comment:
I don't remember why I changed these. But then again probably unrealistic to assume 0 error/variance. Should I revert?